dashboard/.streamlit/config.toml

```diff
@@ -0,0 +1,7 @@
+[theme]
+base = "dark"
+primaryColor = "#6a0dad"  # Purple primary color
+backgroundColor = "#121212" # Dark background color
```

Comment:
Gamers love dark themes 😍

Reply:
Agreed. Well done catering to target audience.